internal/testing/conformance/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh -x
```

Comment:
Curious for thoughts: is it valuable to keep this script around as a "how to run this from your machine" type thing, or would it be better to kill the script here and write up something somewhere else?

Reply:
I could see some value in being able to run this locally, but given that so much logic is interleaved in GitHub Actions commands, not sure if there's a sustainable way to do this that doesn't copypasta everything - the on-demand runner (can that be triggered on any branch/PR?) is probably sufficient, I'm curious if any tool like https://github.com/nektos/act would run this properly locally though.